1.4.0

Foulest opened the pull request:
Fixed extension not working with Twitter's rebrand to X by adding x.com domains to the manifest file.

Foulest committed:
Added X URLs to manifest

Foulest committed:
Merge pull request #1 from Foulest/Foulest-patch-1

Added X URLs to manifest

Foulest commented:
This also fixes #2.

Foulest committed:
Create ci.yml

Foulest committed:
Update ci.yml

Foulest committed:
Fix React issue

Foulest committed:
Update index.tsx

Foulest committed:
Fix React issue

Foulest committed:
Update README.md

Foulest committed:
Update CI

Foulest committed:
Update README.md

Foulest committed:
Create LICENSE

Foulest committed:
Merge branch 'main' of https://github.com/Foulest/blue-check-hider

Foulest committed:
Update workflows

Foulest committed:
Optimize icons

Foulest committed:
Add documentation

Foulest committed:
Update README.md

Foulest committed:
Update .gitignore

Foulest committed:
Update build.yml

Foulest committed:
Update README.md

Foulest committed:
Delete codeql.yml

Foulest committed:
Update manifest.json

Foulest committed:
Update README.md

Foulest committed:
Create package-lock.json

Foulest committed:
Update SECURITY.md

Foulest committed:
Update SECURITY.md

Foulest committed:
Update pull_request_template.md

Foulest committed:
Update documentation

Foulest committed:
Update CODE_OF_CONDUCT.md

Foulest committed:
Update documentation

Foulest committed:
Remove license

Foulest committed:
Add MIT license

Foulest committed:
Update README.md

gsajith commented:
Awesome, thanks for the update and also all the template files etc 😅

Let me know once you're done with changes and I'm happy to merge this.

Foulest commented:
I'll get it ready to merge!

Foulest committed:
Update README.md

Foulest committed:
Cleanup code

- Remove ESLint from project; using JetBrains linting
- Cleaned up project code
- Testing build with artifact

Foulest committed:
Update build.yml

Foulest committed:
Update manifest.json

Foulest committed:
Update documentation

Foulest committed:
Fixed deprecation issue

Foulest committed:
Commented out function

Foulest committed:
Fixed response TODO

Foulest committed:
Fixed magic number

Foulest committed:
Fixed code formatting

Foulest committed:
Update CODE_OF_CONDUCT.md

Foulest committed:
Fix language on button

Foulest committed:
Remove response output

Foulest committed:
Update .gitignore

Foulest committed:
Update dependencies

Foulest committed:
Delete react-app-env.d.ts

Foulest committed:
Update reportWebVitals.ts

Foulest commented:
Done. Updated the codebase, all dependencies, documentation, etc. Builds work! Version is now 1.4.0.

Foulest commented:
@gsajith You can review changes and merge.

Depact commented on src/components/Accordion.tsx:
```diff
@@ -1,58 +1,58 @@
+    width: 100%;
+    margin-top: 6px;
+    border-radius: 6px;
+    color: rgb(0, 0, 0);
```
Is there any advantage to this change?

Foulest replied:
Using RGB values is generally better than using hex, but it's just preference.

Foulest committed:
Update CONTRIBUTING.md

Foulest committed:
Update SECURITY.md

Foulest committed:
Update documentation

Foulest committed:
Add cross-env to fix building

Foulest committed:
Update @types/jest

Foulest committed:
Update .gitignore

Foulest committed:
Add semicolons to line ends

Foulest committed:
Added feature detection for CSS

Foulest committed:
Utilize MutationObserver for performance

Foulest committed:
Handle undefined case in callback

Foulest committed:
Update package.json

Foulest commented:
Push `6fe58af` massively improves performance!

Foulest committed:
Update variable names

Foulest committed:
Update README.md

Foulest committed:
Fixed unresolved error

Foulest committed:
Optimize storage usage
